Exercise_A/index.html

```diff
@@ -1,45 +1,270 @@
+              </div>
+            </div>
+            <div class="col-lg-3 col-md-6 text-center">
+              <div class="service-box">
+                <i class="fa fa-4x fa-heart wow bounceIn text-1" data-wow-delay=".3s"
+                  style="visibility: visible; animation-delay: 0.3s; animation-name: bounceIn;"></i>
+                <h3>Made with Love</h3>
+                <p class="text-muted">Because nothing says hard work and talent like editing a few lines of text.</p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
     <!-- Four pictures with overlay etc -->
+    <section id="portfolio" class="p-0">
+      <div class="container-fluid">
+        <div class="row no-gutter">
+          <div class="col-lg-3 col-sm-6">
+            <div class="porfolio-box">
+              <img class="img-fluid" src="/Exercise_A/assets/images/1.jpg" alt="">
+              <div class="portfolio-box-caption">
+                <div class="portfolio-box-caption-content">
+                  <div class="project-category text-faded">
+                    One Thing
+                  </div>
+                  <div class="project-name">
+                    Look
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="col-lg-3  col-sm-6">
+            <div class="porfolio-box">
+              <img class="img-fluid" src="/Exercise_A/assets/images/2.jpg" alt="">
+              <div class="portfolio-box-caption">
+                <div class="portfolio-box-caption-content">
+                  <div class="project-category text-faded">
+                    Two Thing
+                  </div>
+                  <div class="project-name">
+                    At
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="col-lg-3 col-sm-6">
+            <div class="porfolio-box">
+              <img class="img-fluid" src="/Exercise_A/assets/images/3.jpg" alt="">
+              <div class="portfolio-box-caption">
+                <div class="portfolio-box-caption-content">
+                  <div class="project-category text-faded">
+                    Three Thing
+                  </div>
+                  <div class="project-name">
+                    This
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+          <div class="col-lg-3 col-sm-6">
+            <div class="porfolio-box">
+              <img class="img-fluid" src="/Exercise_A/assets/images/4.jpg" alt="">
+              <div class="portfolio-box-caption">
+                <div class="portfolio-box-caption-content">
+                  <div class="project-category text-faded">
+                    Four Thing
+                  </div>
+                  <div class="project-name">
+                    Bullshit
+                  </div>

```

Comment:
Great nested code and implementation of the four pictures in terms of columns and the overlay.